lib/propolis/src/cpuid.rs

```diff
@@ -116,7 +116,7 @@ impl From<Set> for Vec<bhyve_api::vcpu_cpuid_entry> {
     }
 }
 
-/// Convert a [vcpu_cpuid_entry](bhyve_api::vcpu_cpuid_entry) into an ([Ident],
+/// Convert a [vcpu_cpuid_entry] into an ([Ident],
```

Comment:
Ok, there are enough of these changes that I'm curious. Did you do them manually, or did you run some `cargo doc` command that fixed them?

Reply:
They threw doc errors on `cargo doc` run of CI.

Reply:
I'll push these out into a separate PR.